shims/src/main/java/org/roaringbitmap/ArraysShim.java

```diff
@@ -48,17 +48,11 @@ public static int mismatch(byte[] a, int aFromIndex, int aToIndex,
     int aLength = aToIndex - aFromIndex;
     int bLength = bToIndex - bFromIndex;
     int length = Math.min(aLength, bLength);
-    int i = 0;
-    boolean foundMismatch = false;
-    for (; i < length; i++) {
+    for (int i = 0; i < length; i++) {
```

Comment:
The new code is not equivalent to the previous code. If the lengths differ, the previous code would never return -1, whereas it seems that the new code would.

Reply:
I'll merge and fix.